src/sass/partials/global/_mixins.scss

```diff
@@ -1,15 +1,31 @@
-@mixin button ($border-color, $color, $border-color-hover, $color-hover) {
-  background-color: setting-get('white');
-  border-color: $border-color;
+@mixin button ($background, $border, $color, $background-hover, $border-hover, $color-hover, $shadow) {
```

Comment:
I thought we had discussed calculating the hover color using tint and shade or the IBM Colors [`color-tint` and `color-shade`](https://github.com/ibm-design/colors#color-tint)